### DISCUSSION TITLE
Error: ENOENT: no such file or directory when running example with swissqrbill/pdf in Next.js 14

### DISCUSSION THREAD

**Discussion by riculum** [schoero/SwissQRBill#446] [Q&A]
I'm encountering an error when trying to run a simple example of generating a PDF with swissqrbill/pdf in my Next.js 14 app. The error message is as follows:
`Error: ENOENT: no such file or directory, open '/home/xxxxx/xxxxx/.next/server/vendor-chunks/data/Helvetica.afm'`

```
import { createWriteStream } from "fs";

import PDFDocument from "pdfkit";
import { SwissQRBill } from "swissqrbill/pdf";

import { data } from "./data.js";


const stream = createWriteStream("./output/swissqrbill.pdf");
const pdf = new PDFDocument();
const qrBill = new SwissQRBill(data);

qrBill.attachTo(pdf);

pdf.pipe(stream);
pdf.end();
```

I am running Next.js 14 on a ubuntu client.

Is there a configuration or workaround required to ensure Helvetica.afm is accessible, or should pdfkit or swissqrbill/pdf be bundled differently in Next.js 14? Any advice on resolving this issue would be appreciated!

**Comment by schoero** [comment]
When used on the backend with Node.js, PDFKit should provide the `.afm` font files for you. You may need to tell Next.js or your bundler that PDFKit runs on the backend. 

Have a look at the PDFKit repository: https://github.com/foliojs/pdfkit/issues?q=.afm. Maybe [this could be a solution](https://github.com/foliojs/pdfkit/issues/1549#issuecomment-2387746558) that works for you.
